src/services/qrService.ts

```diff
@@ -476,13 +471,13 @@ const connectOpenId4VcIssuance = async (args: IQrDataArgs): Promise<void> => {
                 },
               },
             ],
-            onDecline: async () => {
+            onDecline: async (): Promise<void> => {
               // TODO WAL-541 fix navigation hierarchy
-              args.navigation.navigate(ScreenRoutesEnum.QR_READER, {})
+              args.navigation.navigate(ScreenRoutesEnum.QR_READER, {});
```

Comment:
Does this mean that when the user is using a deeplink and declines, they will go to the QR code reader?

IMO this should go to the list credentials/home screen anyway. Even in a non-deeplink use case: If I scan a QR code then decline the credential, I would not expect to get back to the QR code scanner. The QR either is gone now, or it is still the same QR. Going to the overview screen simply makes more sense and is more natural

Reply:
changed navigation to go to credentials overview